google-cloud/CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 # Release History
 
+### 0.61.0 / 2019-08-20
+
+#### Features
```

Comment:
I think this should be the following instead:

* Update google-cloud-firestore dependency to 1.0